packages/atomic/src/components/result-template-components/atomic-result-sections/atomic-result-section-actions.tsx

```diff
@@ -0,0 +1,26 @@
+import {Element, Component, State} from '@stencil/core';
+import {
+  InitializableComponent,
+  Bindings,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {containsVisualElement} from '../../../utils/utils';
+
+/**
+ * The `atomic-result-section-actions` element, when added to a result template,
+ * changes the style and position of its content to match specifications
+ * from the result list element.
+ */
+@Component({
+  tag: 'atomic-result-section-actions',
+  shadow: false,
+})
+export class AtomicResultSectionActions implements InitializableComponent {
+  @InitializeBindings() public bindings!: Bindings;
```

Comment:
I don't see why these components need to extend InitializableComponent and use bindings, error.
Just the host is sufficient